Mathlib/ModelTheory/Bundled.lean

```diff
@@ -198,6 +259,47 @@ instance ElementarySubstructure.toModel.instSmall {M : T.ModelType}
     (S : L.ElementarySubstructure M) [h : Small.{w, x} S] : Small.{w, x} (S.toModel T) :=
   h
 
+variable {M N P : CategoryTheory.Bundled.{w} L.Structure}
+
+namespace Embedding
+
+@[simp]
+theorem eqToHom_comp (h : M = N) (h' : N = P) :
+    (eqToHom h').comp (eqToHom h) = eqToHom (h.trans h') := eqToHom_trans h h'
```

Comment:
```suggestion
theorem eqToHom_comp_eqToHom (h : M = N) (h' : N = P) :
    (eqToHom h').comp (eqToHom h) = eqToHom (h.trans h') := eqToHom_trans h h'
```
otherwise your reader might expect the reverse equality